backends/p4tools/common/lib/table_utils.cpp

```diff
@@ -0,0 +1,82 @@
+#include "backends/p4tools/common/lib/table_utils.h"
```

Comment:
no copyright?

Reply:
None of the p4tools file have a copyright, but it is a good point. Something we should add.